packages/netlify-cms-core/src/components/Collection/Entries/Entries.js

```diff
@@ -1,10 +1,21 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from '@emotion/styled';
 import ImmutablePropTypes from 'react-immutable-proptypes';
 import { translate } from 'react-polyglot';
-import { Loader } from 'netlify-cms-ui-default';
+import { Loader, lengths } from 'netlify-cms-ui-default';
 import EntryListing from './EntryListing';
 
+const PaginationMessage = styled.div`
+  width: ${lengths.topCardWidth};
+  padding: 16px;
+  text-align: center;
+`;
+
+const NoEntriesMessage = styled(PaginationMessage)`
+  margin-top: 16px;
```

Comment:
Removed the card background here, felt strange to see what appeared to be an entry showing that message.

Reply:
I actually used the card since to me it looks like the text is "floating" without it:
![image](https://user-images.githubusercontent.com/26760571/78131862-6af04f80-7424-11ea-9b90-54ffd53ce062.png)

Reply:
Yeah that's true - maybe we can find a different way? We've taught the user that those entry cards care clickable, so seeing it for a static message was confusing.

Reply:
Yeah, not a bit issue - we can do it in another PR